samples/boards/nrf/nrf2_clock_control/configs/fll16m.conf

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) 2024 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_CLOCK_CONTROL_NRF2=y
+
+CONFIG_SAMPLE_CLOCK_FREQUENCY_HZ=16000000
```

Comment:
Why request any value here when there is anyway no other choice than 16 MHz? Similarly for `lfclk`, it is always 32768. And for UART, users don't actually need to know/care what is the frequency of the clock that drives it.

Reply:
removed frequency (now uses default 0)